libs/books/feature/src/lib/book-search/book-grid/book-grid.component.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SharedTestingModule } from '@tmo/shared/testing';
+import { BookGridComponent } from './book-grid.component';
+import { BooksFeatureModule } from '../../books-feature.module';
+
+describe('BookGridComponent', () => {
+  let component: BookGridComponent;
+  let fixture: ComponentFixture<BookGridComponent>;
+  const books = [{
+    id: `${Date.now()}a`,
```

Comment:
As with the earlier comment, there's no advantage to using random(ish) ids or titles here. Why not just pick some readable titles and stick with them? You can even choose some real author's names! Unless there's a reason to generate random values, have some fun with test data. Pick books that you love. It will make the output of the tests if they fail much more readable.